UI/src/view/linear_asset/linearAssetLayer.js

```diff
@@ -178,7 +178,15 @@ root.LinearAssetLayer  = function(params) {
   });
   map.addLayer(me.indicatorLayer);
   me.indicatorLayer.setVisible(false);
-  this.readOnlyLayer = new GroupedLinearAssetLayer(params, map);
+  this.readOnlyLayer = massLimitation ? new GroupedLinearAssetLayer(params, map) : {
```

Comment:
Onko this.readOnlyLayer tarpeellinen vain massarajoituksille? Eikö tämä riko muilla tietolajeilla toimintaa?

Reply:
Ainakaan omien testailujen perusteella ei rikkonut. Noihin lineaaristen readOnlyassettien hakuun on kovakoodattu massarajoitusten endpoint eli toi layer käsittelee joka tapauksessa massarajoituksia.